Ginger/GingerCore/Actions/UIAutomation/UIAComWrapperHelper.cs

```diff
@@ -434,6 +434,12 @@ public override List<AppWindow> GetListOfDriverAppWindows()
                         {
                             continue;
                         }
+
+                        Process isLocalProcess = Process.GetProcesses().FirstOrDefault(p => p.StartInfo.Environment["USERNAME"] == Environment.UserName);
```

Comment:
Why we are iterating all the processes again. We just need to if process with id window.Current.ProcessId belongs to current user or not. Like we did for java